README.md

```diff
@@ -36,15 +36,19 @@ reader = IonReaderBuilder.Build(text, new ReaderOptions {Catalog = catalog});
     text: "hello world"
 }
 */
-Console.WriteLine(reader.MoveNext()); // Struct
-reader.StepIn();
-Console.WriteLine(reader.MoveNext()); // Int
-Console.WriteLine(reader.CurrentFieldName); // "number"
-Console.WriteLine(reader.IntValue());   // 1
-Console.WriteLine(reader.MoveNext()); // String
-Console.WriteLine(reader.CurrentFieldName); // "text"
-Console.WriteLine(reader.StringValue());   // "hello world"
-reader.StepOut();
+
+using (reader)
```

Comment:
Can we instantiate the reader in the using block,

Dotnet says its bad to instantiate it outside of the using block

> You can instantiate the resource object and then pass the variable to the using statement, but this is not a best practice. In this case, after control leaves the using block, the object remains in scope but probably has no access to its unmanaged resources. In other words, it's not fully initialized anymore. If you try to use the object outside the using block, you risk causing an exception to be thrown. For this reason, it's generally better to instantiate the object in the using statement and limit its scope to the using block.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement

Reply:
We should probably be consistent with the code, in which "using" is used with the braces. And just in case with compatibility as well? The other form is only with C# 8.0 and above.